surface/include/pcl/surface/ball_pivoting.h

```diff
@@ -214,8 +305,10 @@ namespace pcl
      * @param num_point_in_radius
      * @param ratio_success
      */
-    virtual void
-    setEstimatedRadius (const int num_sample_point, const int num_point_in_radius, const float ratio_success);
+    void
+    setSearchRadiusAutomatically (const int num_sample_point = 0, 
```

Comment:
should not be, now non-positive parameter would be changed to one ratio of point cloud size, 10% or 20% i remember